spectral/ruleset.yml

```diff
@@ -477,20 +458,6 @@ rules:
     then:
       - field: '500'
         function: truthy
-  responses-must-have-content-type:
```

Comment:
I think this was a rule carried over from before. Not necessary and is being checked properly by a different rule